app/Services/ProductionRequestActivityLogger.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Services;
+
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+
+class ProductionRequestActivityLogger extends AbstractRequestActivityLogger
+{
+    protected function collectRequestData(Request $request): array
+    {
+        return ["Url" => $request->fullUrl() ,"Time" => Carbon::now()];
```

Comment:
I propose to include in all your loggers the ip, it provides value in long run.

Reply:
Added IP to Debug and Production loggers.